public/vendor/flot/jquery.flot.js

```diff
@@ -3052,6 +3054,42 @@ Licensed under the MIT license.
           triggerClickHoverEvent("plotclick", e, function (s) { return s["clickable"] != false; });
         }
 
+        function onTouch(e) {
```

Comment:
Would be good to annotate this code with `// grafana addition` as we do for vendor code modifications :)